pages/glossary/adequate-inspection-state.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Adequate inspection state
+key: adequate-inspection-state
+unambiguous: true
+objective: true
+---
+
+An element is in an _adequate inspection state_ when it is in any of the [inspection states][] except the state [focused][] and [hovered][].
```

Comment:
I'm kinda stumped by this definition. What is this for?

Reply:
To make it easier to read the expectation. Compare 

>For each [link history state][] of the [semantic link][] there exists at least one [adequate inspection state][] and 

with 

> For each [link history state][] of the [semantic link][] when the link is not in the focused state and the hovered state at the same time and

Reply:
To be honest, I personally much prefer the latter version as it was immediately obvious what was meant. The former phrasing does seem like it adds a confusing layer of indirection.

Reply:
Alright. Given the support, I've rephrased the expectation.